agent/stats/utils_unix_test.go

```diff
@@ -40,3 +40,33 @@ func TestDockerStatsToContainerStatsZeroCoresGeneratesError(t *testing.T) {
 		t.Error("Expected error converting container stats with empty PercpuUsage")
 	}
 }
+
+func TestDockerStatsToContainerStatsCpuUsage(t *testing.T) {
+	// doing this with json makes me sad, but is the easiest way to deal with
+	// the inner structs
+
+	// numCores is a global variable in package agent/stas
+	// whichdenotes the number of cpu cores
+	numCores = 4
+	jsonStat := fmt.Sprintf(`
+		{
+			"cpu_stats":{
+				"cpu_usage":{
+					"percpu_usage":[%d, %d, %d, %d],
+					"total_usage":%d
+				}
+			}
+		}`, 1, 2, 3, 4, 100)
+	dockerStat := &docker.Stats{}
+	json.Unmarshal([]byte(jsonStat), dockerStat)
+	containerStats, err := dockerStatsToContainerStats(dockerStat)
+	if err != nil {
```

Comment:
can you switch to `assert`'s and `require`'s in this refactor (here and in the other test as well)